app/views/landing_page/blocks/_featured.html.erb

```diff
@@ -9,7 +9,7 @@
     <% block.featured_content.each do |subblock| %>
       <%
         options = {}
-        options[:position] = "top" if subblock.type == "heading"
```

Comment:
Do we want to have a large heading size in the Featured block as well? I'm guessing this was already here and you're just updating it to match, but I think if the hero already has a large heading, we probably don't need another one?

Reply:
Yeah, we're not actually using the featured block (I don't think?), so it's a bit of a moot point. I'd be happy to go either way on it. Shall I just remove the option?

Reply:
Yeah let's remove it entirely from the featured block, thanks 👍

Reply:
Done in d44c687